dvc/commands/experiments/__init__.py

```diff
@@ -58,3 +58,42 @@ def add_parser(subparsers, parent_parser):
     for cmd in SUB_COMMANDS:
         cmd.add_parser(experiments_subparsers, parent_parser)
     fix_plumbing_subparsers(experiments_subparsers)
+
+
+def add_rev_selection_flags(
+    experiments_subcmd_parser, command: str, default: bool = True
+):
+    experiments_subcmd_parser.add_argument(
+        "-A",
+        "--all-commits",
+        action="store_true",
+        default=False,
+        help=(
+            f"{command} all experiments in the repository "
+            "(overrides `--rev`and `--num`)."
+        ),
+    )
+    default_msg = "(HEAD by default)" if default else ""
+    msg = (
+        f"{command} experiments derived from the specified `<commit>` as "
+        f"baseline{default_msg}."
```

Comment:
Minor: missing a space I think 🙂 
```suggestion
    default_msg = " (HEAD by default)" if default else ""
    msg = (
        f"{command} experiments derived from the specified `<commit>` as "
        f"baseline{default_msg}."
```

Reply:
But... Will it be clear that ``from the specified `<commit>` `` refers to using `--rev` ?

Reply:
UPDATE: Nvmd on that question, I see this IS the help output for --rev 😁